sdk/storage_blobs/src/options/block_id.rs

```diff
@@ -1,13 +1,25 @@
 use azure_core::AppendToUrlQuery;
 use bytes::Bytes;
 
+/// Struct wrapping the bytes of a block blob block-id,
+///
+/// A block id cannot exceed 64 bytes. In addition all block id's in a block list must be the same length when converted into a base64 string.
```

Comment:
Including these restrictions in the documentation is a good idea.  Can we link to a citation on learn.microsoft.com for this?

Reply:
Added  a link to documentation